apps/framework-docs/src/pages/moose/olap/model-materialized-view.mdx

```diff
@@ -13,7 +13,27 @@ import { Lightbulb } from "lucide-react";
 
 ## Overview
 
-Materialized views are write-time pipelines. They run a static `SELECT` to transform data from one or more source table(s) to a target destination table. You query the destination table like any other table.
+Materialized views are write-time pipelines. They run a static `SELECT` to transform data from one or more source table(s) to a target destination table. You query the destination table like any other table. The MaterializedView class wraps ClickHouse's `MATERIALIZED VIEW` functionality, letting you model these transformations in code rather than raw SQL.
+
+<BulletPointsCard title="Key differences from other databases" bulletStyle="number" compact bullets={[
+    "Requires both a transformation query AND a target table definition",
+    "Automatically handles backfilling historical data",
+    "Manages dependencies between multiple materialized views"
+]} />
+
+<BulletPointsCard title="What Moose handles for you"
+bulletStyle="check"
+compact
+bullets={[
+    "Orchestrates the transformation query and target table setup",
```

Comment:
is it clear to a reader what orchestrating means here, and why its valuable? might want to be more explicit? 
- only write one sql query
- don't have to execute anything
(i think?)